apiserver/plane/app/views/view/base.py

```
@@ -344,12 +336,32 @@
             if not value or key not in customPropertyAllowedKeys:
                 continue
-            groupedCustomProperties.setdefault(key, set()).add(value)
 
-        groupedUniqueCustomProperties = {key: list(values) for key, values in groupedCustomProperties.items()}
-        unique_values_list = [{"key": k, "values": v} for k, v in groupedUniqueCustomProperties.items()]
+            bucket = groupedCustomProperties.setdefault(
+                key, {"values": set(), "data_type": data_type}
+            )
+            bucket["values"].add(value)
+
+        groupedUniqueCustomProperties = {
```

Pylint (3.3.7)

[convention] Variable name "groupedUniqueCustomProperties" doesn't conform to snake_case naming style

(C0103)